odoo/addons/base/models/res_partner.py

```diff
@@ -644,6 +644,7 @@ def write(self, vals):
 
     @api.model_create_multi
     def create(self, vals_list):
+        vals_list = [{k: v.replace('@','') if k == 'name' else v for k, v in vals.items()} for vals in vals_list]
```

Comment:
check your linter
```suggestion
        vals_list = [{k: v.replace('@', '') if k == 'name' else v for k, v in vals.items()} for vals in vals_list]
```

Reply:
Don't do that. People should be able to put whatever they want as partner name. Purpose here is to be defensive (aka remove @ in names when sending emails), but it is a valid name if you really want it.